keras_trainer/trainer.py

```diff
@@ -41,6 +41,8 @@ class Trainer(object):
         'momentum': {'type': float, 'default': 0.9},
         'sgd_lr': {'type': float, 'default': 0.01},
         'pooling': {'type': str, 'default': 'avg'},
+        'sigmoid': {'type': bool, 'default': False},
```

Comment:
Maybe better to have `activation` which can be `'sigmoid'` or `'softmax'`

Reply:
True